source-salesforce/test.flow.yaml

```diff
@@ -0,0 +1,4819 @@
+---
+captures:
+  acmeCo/source-salesforce:
+    endpoint:
+      local:
+        command:
+          - python
+          # - "-Xfrozen_modules=off"
+          # - "-m"
+          # - debugpy
+          # - "--listen"
+          # - "0.0.0.0:5678"
+          # - "--wait-for-client"
+          - "-m"
+          - source-salesforce
+        config: config.yaml
+    bindings:
```

Comment:
Please remove these, leaving just `bindings: []`.

They're already covered by the discovery snapshot.

Reply:
modified in [ae13497](https://github.com/estuary/connectors/pull/1355/commits/ae134972e57190b963389f75daf5e7d6a1bf6d92)

Reply:
updated in [a1c34e9](https://github.com/estuary/connectors/pull/1355/commits/a1c34e924c00ab8a950e247e7bff669e8fcd0abd)